### DISCUSSION TITLE
Refreshing the Page, all Details are getting erased!

### DISCUSSION THREAD

**Discussion by sayakongit** [github/india#261] [Students]
I have submitted several times my details and all the links, still it is getting removed every time I refresh the page or re-login.  Kindly look into this matter.

**Comment by Noronhajoel97** [comment]
Don't worry,
1. just clear your browsing data on your browser
2. Then open the login page in incognito mode and fill up everything again.

**Reply by TheSuranaverse** [reply]
**FUTURE READY SEPARATE CHANNEL**

For smooth communication & discussion!
Link: https://discord.gg/ceMXzhfaka

Thank you 🙏

**Comment by Kamaruddheen** [comment]
> I have submitted several times my details and all the links, still it is getting removed every time I refresh the page or re-login. Kindly look into this matter.

Yeah man, looks like they purposely disabled all the features in Google Chrome except other browsers (Firefox, Edge) I guess so. I tried on Firefox and Edge browser. It is working fine.

**Reply by TheSuranaverse** [reply]
**FUTURE READY SEPARATE CHANNEL**

For smooth communication & discussion!
Link: https://discord.gg/ceMXzhfaka

Thank you 🙏

**Comment by TheSuranaverse** [comment]
**FUTURE READY SEPARATE CHANNEL**

For smooth communication & discussion!
Link: https://discord.gg/ceMXzhfaka

Thank you 🙏
